api4/plugin.go

```diff
@@ -141,6 +141,13 @@ func installPluginFromUrl(c *Context, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	// Store bundle in the file store to allow access from other servers.
```

Comment:
Good catch for adding this block too, but now that we know there's multiple entry points, I wonder if we shouldn't just push this into `c.App.InstallPlugin`. In particular, it looks like it's possible to install a plugin from the CLI which just invokes `InstallPlugin` directly but then wouldn't trigger this new code.

Reply:
Agreed. I think it's more robust to have it as the last step of InstallPlugin within App. The reason I left it out was do to some details handling the `io.Reader` interface and reseting after it was fully read in order to write the file bundle. I'll take a second look and reconsider this which makes even more sense now.

Reply:
Yeah, you'll need to upgrade the `io.Reader` to be an `io.ReadSeeker`, but I think that should be straightforward.

Reply:
✅Fixed